gulpfile.js

```diff
@@ -57,31 +71,61 @@ const cleanDist = (cb) => {
 };
 cleanDist.displayName = '🧹 cleaning up dist path';
 
+/**
+ * Copy Sass layout files from design-system-tokens to theme styles folder
+ */
+
+const copySassLayoutTokens = (cb) => {
+  const sassLayoutTokensFiles = `${tokensPackageFiles}/css-vars/${theme()}-layout-tokens.scss`;
+
+  gulp
+    .src(sassLayoutTokensFiles)
+    .pipe(rename('_layout.scss'))
+    .pipe(gulp.dest(path.join(srcPath, 'styles')))
+    .on('end', cb);
+};
+
+copySassLayoutTokens.displayName = '📎 copying layout tokens to src/styles folder';
+
 /**
  * Copy theme files from styles/themes to dist
  */
 const copyThemes = (cb) => {
-  const themeFiles = `${srcPath}/styles/*-theme.css`;
+  const tokensFiles = `${tokensPackageFiles}/css-vars/${theme()}-theme.css`;
 
   gulp
-    .src(themeFiles)
+    .src(tokensFiles)
     .pipe(gulp.dest(path.join(distPath, 'css')))
     .on('end', cb);
 };
 
 copyThemes.displayName = '📎 copying themes to dist/css folder';
+
+/**
+ * Copy theme files into docs
+ */
+const copyThemesToDocs = (cb) => {
+  const tokensFiles = `${tokensPackageFiles}/css-vars/*-theme.css`;
+
+  gulp
+    .src(tokensFiles)
+    .pipe(gulp.dest(path.join('packages', 'docs', 'static', 'themes')))
+    .on('end', cb);
+};
+
+copyThemesToDocs.displayName = '📎 copying themes to docs folder';
+
 /**
  * compile sass assets to css, copy to /dist/css folder
  */
 const compileSass = (cb) => {
   const envDev = process.env.NODE_ENV === 'development';
 
-  const sassSourcePaths = isCore
-    ? `${srcPath}/styles/**/*.scss`
-    : [`${sassCorePath}/**/*.scss`, `${srcPath}/styles/**/*.scss`];
+  const sassSourcePaths = `${tokensPackageFiles}/scss/${theme()}-*.scss`;
```

Comment:
This is no longer compiling the sass of each design system package. It is only compiling the sass variables files in the tokens. That is, we no longer get a `dist/css/index.css` in each of the design systems.

Reply:
I don't think we actually need any of the changes in this function, but I could be wrong.

Reply:
i was trying to move all the logic out of the copy_themes.sh file. in there, there was logic to copy the Sass files into dist/sass, while our gulp func seemed to compile Sass files into CSS (dist/css). 

am i good to just omit copying our Sass to dist/sass?

Reply:
Oh I see, the `tokens/dist/scss/*` files still need to be copied to our `[ds-packages]/dist/scss` folder. This function is compiling them, though, so I think we could just add those scss files to the glob in `copyThemes`, since that is just a file copy.

Reply:
i'm rolling back this func to what it was before and breaking the scss copy logic out into its own func. 

i tried grouping it with the `copyThemes` func, but ran into build errors. mainly because i was trying to take files from 2 different sources (`tokens/**/css-vars/*.css` and `tokens/**/scss/*.scss`) and copy them into different directories (`theme/dist/css/*` and `theme/dist/scss/*`). gulp couldn't handle that logic in a singular function